app/models/theme_setting.rb

```diff
@@ -54,7 +54,7 @@ def json_value_size
         :json_value,
         I18n.t(
           "theme_settings.errors.json_value.too_large",
-          max_size_megabytes: MAXIMUM_JSON_VALUE_SIZE_BYTES / 1024 / 1024,
+          max_size: MAXIMUM_JSON_VALUE_SIZE_BYTES / 1024 / 1024,
```

Comment:
I think we can pull this fix out into another PR since this is not directly related to the upgrading Rails.

Reply:
Done in https://github.com/discourse/discourse/pull/27450